sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -33,9 +33,8 @@
 
 
 class StaticFeesReqHandler(FeeReqHandler):
-    valid_txn_types = {SET_FEES, GET_FEES, FEE_TXN}
-    write_types = {SET_FEES, FEE_TXN}
-    query_types = {GET_FEES, }
+    write_types = FeeReqHandler.write_types.union({SET_FEES, FEE_TXN})
```

Comment:
Ok, so, it looks like the problems are because IndyNode already has `ConfigReqHandler` registered to process some Config txn types, see `register_req_handler` and `register_txn_type` in plenum's `node.py`.
The Plugin tries to override it with  `StaticFeesReqHandler` (using `register_req_handler` in `main.py`), so it calls `register_txn_type` for already registered txns, which causes ValueError.

One of the options is to created  to unregister `unregister_req_handler` in plenum's node, or to enhance `register_req_handler` to unregister old req handler when a new one overrides it.